db_access.py

```diff
@@ -30,13 +33,25 @@ def delete_user(user_id):
     conn.commit()
 
 
-def add_user(handle):
+def add_user(handle, password):
+    password = hash_ps(password)
     conn = get_db()
-    conn.execute('INSERT INTO user (handle, admin) values (:handle, :admin)',
-                 {'handle': handle, 'admin': 0})
+    conn.execute('INSERT INTO user (handle, password, admin) values (:handle, :password, :admin)',
+                 {'handle': handle, 'password':password, 'admin': 0})
     conn.commit()
 
 
+def create_account(handle, password):
+    conn = get_db()
+    res = conn.execute('SELECT * FROM user WHERE handle=:handle', {'handle':handle})
+    if list(res):
+        state = False
```

Comment:
I think keeping the flash in the chirpr.py file is the best way to go. Everything flash in one file.